src/common/sorted-table/index.js

```diff
@@ -134,7 +143,40 @@ class ColumnHead extends Component {
 
 // ===================================================================
 
+@propTypes({
+  indeterminate: propTypes.bool.isRequired
+})
+class Checkbox extends Component {
+  componentDidUpdate () {
+    const { props: { indeterminate }, ref } = this
+    if (ref !== null) {
+      ref.indeterminate = indeterminate
+    }
+  }
+
+  _ref = ref => {
+    this.ref = ref
+    this.componentDidUpdate()
+  }
+
+  render () {
+    const { indeterminate, ...props } = this.props
+    props.ref = this._ref
+    props.type = 'checkbox'
+    return <input {...props} />
+  }
+}
+
+// ===================================================================
+
 const DEFAULT_ITEMS_PER_PAGE = 10
+const actionsShape = propTypes.arrayOf(propTypes.shape({
+  // function called with the selected items in parameters
```

Comment:
I believe this comment is incorrect.

And the diff between `groupedActions` and `individualActions` should be explicited.